Corgibytes.Freshli.Cli/Commands/MainCommand.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.CommandLine;
+
+namespace Corgibytes.Freshli.Cli.Commands
+{
+    public class MainCommand : RootCommand
+    {
+        public MainCommand() : base("Root Command")  { }
+    }
+}
```

Comment:
I created the main command because the library we use needs a root command. If you see this inherits from the RootCommand and the ScanCommand and Auth command inherits from  the Command class.

It is used in the program.cs file. Another option is to just do "new RootCommand()" there but I wanted to be consistent and also this way we can add options/arguments not related to specific commands if needed.

```
 public static CommandLineBuilder CreateCommandLineBuilder()
        {
            CommandLineBuilder builder = new CommandLineBuilder(new MainCommand())
                .UseHost(args =>
                {
                    return CreateHostBuilder(args);
                })
                .UseMiddleware(async (context, next) =>
                {
                    await LogExecution(context, next);
                })
               .UseExceptionHandler()
               .UseHelp();

            foreach (BaseCommand command in Commands)
            {
                builder.AddCommand(command);
            }

            return builder;
        }
```